homeassistant/components/http.py

```diff
@@ -202,6 +202,11 @@ def register_path(self, method, url, callback, require_auth=True):
         """ Regitsters a path wit the server. """
         self.paths.append((method, url, callback, require_auth))
 
+    def log_message(self, fmt, *args):
```

Comment:
I get it now, they are overriding the log methods of the built-in classes. Could we add a comment to not confuse future readers of the code?